core/templates/pages/story-editor-page/modal-templates/new-chapter-title-modal.controller.ts

```diff
@@ -147,8 +152,18 @@ angular.module('oppia').controller('CreateNewChapterModalController', [
                 return;
               }
               $scope.correctnessFeedbackDisabled = false;
-              $scope.updateTitle();
-              $scope.updateExplorationId();
+              ExplorationIdValidationService.categoryNotDefault(
+                $scope.explorationId).then(
+                (categoryNotDefault) => {
+                  if (categoryNotDefault) {
+                    $scope.categoryNotDefault = true;
+                    $scope.$applyAsync();
```

Comment:
Is applyAsync not required in the "else" scenario?

Reply:
I've added this to the else statement as well.